mailbox/backup/src/main/java/org/apache/james/mailbox/backup/zip/package-info.java

```diff
@@ -0,0 +1,74 @@
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * <p>
+ * The zip archive format for the backup of an account
+ * This archive contains the following entries for each mailbox of the user:
+ * <p>
+ * - 'mailboxName/' : directory entry
+ * - 'mailboxName/annotations/' : directory entry, optional. Is present only if the mailbox contains some annotations
+ * - 'mailboxName/annotations/annotation1Key' : file entry which name is the key of the annotation,
+ * the content of the annotation is stored in the content of the file
+ * - 'mailboxName/annotations/annotation2Key' : idem a mailbox may contains several annotations
+ * - 'messageId1' : file entry which name is the serialized id of the message, the content of the message, is stored in the content of the file
+ * - 'messageId2' : a mailbox may contains several messages.
+ * ...
+ * <p>
+ * The order in which the entries are added to the zip file is important and must respect this structure.
+ * This is important because during the restoration the file is read iteratively in memory from an inputstream,
+ * without storing the whole structure on memory nor on disk.
+ * <p>
+ * The zip archive format for the backup of an account
+ * This archive contains the following entries for each mailbox of the user:
+ * <p>
+ * - 'mailboxName/' : directory entry
+ * - 'mailboxName/annotations/' : directory entry, optional. Is present only if the mailbox contains some annotations
+ * - 'mailboxName/annotations/annotation1Key' : file entry which name is the key of the annotation,
+ * the content of the annotation is stored in the content of the file
+ * - 'mailboxName/annotations/annotation2Key' : idem a mailbox may contains several annotations
+ * - 'messageId1' : file entry which name is the serialized id of the message, the content of the message, is stored in the content of the file
+ * - 'messageId2' : a mailbox may contains several messages.
+ * ...
+ * <p>
+ * The order in which the entries are added to the zip file is important and must respect this structure.
+ * This is important because during the restoration the file is read iteratively in memory from an inputstream,
+ * without storing the whole structure on memory nor on disk.
+ *
+ * see org.apache.james.mailbox.backup.ZipperTest unit test for more information about this format
+ */
+
+/**
+ * The zip archive format for the backup of an account
```

Comment:
Duplicate here

Reply:
For this two issues, the reformat command on the package-info file is really messy...